src/security/pages/sign-up.component.vue

```diff
@@ -0,0 +1,44 @@
+<script setup></script>
+
+<template>
+  <div class="flex flex-col items-center mt-8">
+    <div class="py-8">
+      <span class="text-2xl font-semibold"> Create an account </span>
+    </div>
+    <input
+      type="text"
+      class="bg-slate-400 w-100 rounded-lg h-12 placeholder:font-light placeholder:text-white my-4"
+      placeholder="Fullname" />
+    <input
+      type="email"
+      class="bg-slate-400 rounded-lg h-12 placeholder:font-light placeholder:text-white my-4"
+      placeholder="Email" />
+    <input
+      type="password"
+      class="bg-slate-400 rounded-lg h-12 placeholder:font-light placeholder:text-white my-4"
+      placeholder="Password" />
+    <input
+      type="tel"
+      class="bg-slate-400 rounded-lg h-12 placeholder:font-light placeholder:text-white my-4"
+      placeholder="Phone" />
+    <input
+      type="text"
+      class="bg-slate-400 rounded-lg h-12 placeholder:font-light placeholder:text-white my-4"
+      placeholder="Rol" />
+
+    <div class="py-4">
+      <input type="checkbox" />
+      <span clas> Agree with terms and conditions</span>
```

Comment:
Whitespaces again and you left a `class` attribute unused (and it's missing an s).

```suggestion
      <span>Agree with terms and conditions</span>
```